python/config/suites/lgtm/python2-lgtm

```diff
@@ -1,2 +1 @@
-@import "python2-queries-lgtm"
-@import "python2-metrics-lgtm"
+@import "python-lgtm"
```

Comment:
Perhaps add a comment explaining that this file is for backwards compatibility, and can be removed by `(DATE/VERSION)`.